### DISCUSSION TITLE
Problems with popple@22.02.0 on M1 Mac

### DISCUSSION THREAD

**Discussion by rbubley** [Homebrew/discussions#3058] [Everyday usage] [answered]
Hi,

Not sure if this is just my Mac or a more general problem. Poppler binaries (v22.02.0) (e.g. pdfinfo) crash on start for me; console shows a `EXC_BAD_ACCESS (SIGKILL (Code Signature Invalid))`.

Is it working/failing (in a similar environment) for others?

```
HOMEBREW_VERSION: 3.4.1-9-g1447f08
ORIGIN: https://github.com/Homebrew/brew
HEAD: 1447f0853bb2e9006e89d8db0289cfef9e7aa05b
Last commit: 3 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 6e3f2ec6dfc767a0c4855571cc8191fad02ca90f
Core tap last commit: 35 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_CORE_GIT_REMOTE: https://github.com/Homebrew/homebrew-core
HOMEBREW_EDITOR: /usr/local/bin/code
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.8 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 13.0.0 build 1300
Git: 2.35.1 => /opt/homebrew/bin/git
Curl: 7.77.0 => /usr/bin/curl
macOS: 12.2.1-arm64
CLT: 13.2.0.0.1.1638488800
Xcode: N/A
Rosetta 2: false
```

**Comment by SMillerDev** [comment]
How did you install it?

**Reply by rbubley** [reply]
Just with `brew install poppler` (and then after experiencing this problem trying `brew reinstall poppler` - but it made no difference).

**Reply by SMillerDev** [reply]
And how are you starting them?

**Reply by rbubley** [reply]
From a zsh terminal, using e.g. `pdfinfo`

**Comment by carlocab** [comment, marked as answer]
Try `brew reinstall $(brew deps poppler)`.

**Reply by rbubley** [reply]
That cured it, thank you!

**Comment by skratchdot** [comment]
Thank you! I was having this same issue (and even logged an issue in the wrong place hoping someone w/ the same issue knew a fix/workaround: https://gitlab.freedesktop.org/poppler/poppler/-/issues/1226).

```
brew reinstall $(brew deps poppler)
```

Fixed things for me.

Before finding this thread, I was trying to run `brew install --build-from-source poppler` and it kept erroring around something with qt.  When I `brew info qt` I noticed that XCode was no longer installed on my machine (it mysteriously disappeared at some point).  Not sure if that has anything to do with things breaking.

I know last month poppler worked, and I had XCode installed.  I run `brew upgrade` frequently, and I'm not sure exactly when poppler broke, or xcode disappeared 🤷  Anyways, thanks again for this fix!
